t/path.t

```diff
@@ -9,6 +9,7 @@ use Test;
 my $ig = File::Ignore.parse(q:to/LIST/);
     b*z/
     bang
+    /foo
```

Comment:
I think this is a leftover from another change, which I haven't finished. According to git man page `/foo` should ignore the contents of the root directory `foo`, while root is the main git directory.

Currently if I have a repo `/home/user/bar` with a child `foo`. `File::Ignore.ignore-path('/home/user/bar/foo')` will return `False`, because it doesn't maches the `/foo` rule. To fix this `File::Ignore` should probably track what the current root directory is.